docs/essentials/viewport.md

```diff
@@ -47,7 +47,7 @@ The viewports object needs the following keys:
 
 ### Use a detailed set of devices
 
-By default, Storybook uses a [minimal set of viewports](https://github.com/storybookjs/storybook/blob/master/addons/viewport/src/defaults.ts#L135) to get you started. But you're not restricted to these. The addon offers a more granular list of devices that you can use.
+By default, Storybook uses a [minimal set of viewports](https://github.com/storybookjs/storybook/blob/master/addons/viewport/src/defaults.ts#L167) to get you started. But you're not restricted to these. The addon offers a more granular list of devices that you can use.
```

Comment:
@mnigh if you're ok with it, can you update the links for this [one](https://github.com/storybookjs/storybook/blob/next/code/addons/viewport/src/defaults.ts#L167) instead? It would probably be a better option to avoid this getting out of date, potentially leading to an inadvertent 404.